### PR TITLE
Proof-read of the readme

### PR DESCRIPTION
**Pull request by xeviknal**
A little bit of this, a little bit of that.
Spell-check and some small proposal changes. nothing big.
If I have time tomorrow, I'll propose another PR

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -84,25 +91,25 @@ The Car Racing environment features a physics model that affects the behaviour o
...
+* **EarlyStop**: an original wrapper; when used, the environment will output `done = True` in additional circumstances besides the default ones, such as getting a negative average reward. This allows us to stop the execution early and train with more episodes.
 
 There is also the issue of defining what "consistently" means when trying to get a reward of "consistently more than 900 points". We settled on calculating a *running reward* that accumulates the previously obtained rewards and calculates an average of sorts that represents the reward you can expect from the model at a specific stage of training, which is calculated as `running_reward = 0.05 * episode_reward + (1 - 0.05) * running_reward`, where `episode_reward` is the reward we obtain for each lap attempt.
 
 For all of our experiments, the chosen optimizer was Adam, since it seems to be the default optimizer for pretty much any task. We did not experiment with additional optimizers.
```
Comment (xeviknal) [inline README.md:102]: Shouldn't we add a more scientific sentence for the Adam? hahahahaha
Like adding a reference to theory slides where it compares both of them. Or something like that?

**Review by ziritrion — APPROVED** [review]
LGTM. Want me to wait for you to finish your review before merging?

File: README.md (hunk 1 above)
Comment (ziritrion) [reply README.md:102]: I honestly can't think up of anything. Optimizers aren't the focus of this project, so I would just let it be.

File: README.md [hunk 2]
```diff
@@ -84,25 +91,25 @@ The Car Racing environment features a physics model that affects the behaviour o
 We decided to initially approach the task by using policy-based RL methods, starting from REINFORCE-Vanilla Policy Gradient and implementing more sophisticated algorithms as we understood the behavior, advantages and shortcomings of each algorithm. Our chosen library was PyTorch due to our familiriaty with it and its ease of use.
 
 Before implementing any algorithm, however, we knew from our classes and from additional research that using the vanilla environment as-is would be inefficient. The OpenAI Gym framework allows the use of *wrappers*, pieces of code that "wrap" the default environment in order to alter the outputs to make them more convenient for our purposes. Gym already provides some wrappers but we also implemented a few. The wrappers we ended up using were:
-* Monitor: one of Gym's provided wrappers. It provides an easy way to record the output of the environment to a video file.
```
Comment (xeviknal) [inline README.md:87]: Shouldn't we add a video with good results in the video right above?

**Review by ziritrion — APPROVED** [review]

**Pull request merged**